### DISCUSSION TITLE
是否有哪里可以开启：级联保存时，自动删除不存在的子实体？

### DISCUSSION THREAD

**Discussion by netty2019** [dotnetcore/FreeSql#1376] [General]
是否有哪里可以开启：级联保存时，自动删除不存在的子实体？
![image](https://user-images.githubusercontent.com/47128297/207787931-086971dd-32af-4f6d-9d60-0a46c7728a29.png)
用手工开启的方法，好像不支持多级子实体

**Comment by 2881099** [comment]
这个要看下聚合根仓储实验室，文档

**Reply by netty2019** [reply]
你好，是否可以通过在外部增加代码在仓库实现这个功能？谢谢

**Reply by 2881099** [reply]
是说聚合根仓储的功能吗？

**Reply by 2881099** [reply]
普通仓储是第一代功能，2019年，出于可控性的要求，对级联功能支持的比较零散。

聚合根仓储是第二代功能，2022年，自动化级联需求，目前还在实验阶段。

一般来讲，第一代仓储更受欢迎，聚合根仓储可以辅助来使用。
